src/common/paths/buckets_bucketID_labels.yml

```diff
@@ -29,31 +57,100 @@ post:
   tags:
     - Buckets
   summary: Add a label to a bucket
+  description: |
+    Adds a label to a bucket and returns the new label information.
+
+    Labels are objects that contain `labelID`, `name`, `description`, and `color`
+    key-value pairs. They may be used for grouping and filtering InfluxDB
+    resources. Labels are also capable of grouping across different resources.
+    For example, a single label named `air_sensor` can be applied to both a
+    bucket and a task to quickly organize resources.
+
+    Use this endpoint to add a label that you can apply to buckets.
+
+    #### Limitations
+
+    - Before adding a label to a bucket, you must create the label if you have
+    not done so. To create a label, use
+    [Create a label]({{% INFLUXDB_DOCS_URL %}}/api/#operation/PostLabels)
+
+    #### Related guides
+
+    - [/Labels]({{% INFLUXDB_DOCS_URL %}}/api/#tag/Labels)
```

Comment:
```suggestion
```

Reply:
See my other comment about these related endpoints, but the correct syntax here should be:

```
    - [Labels](#tag/Labels)
```

We can use a fragment and avoid the request.